it_zwierzak/homepage/views.py

```diff
@@ -27,6 +28,28 @@ def register(request):
     )
 
 
+def log_in(request):
```

Comment:
zwyczajowo nie ma tego underscore'a
```suggestion
def login(request):
```